src/Controller/PeopleController.php

```diff
@@ -24,19 +24,25 @@ public function index()
     /**
      * @Route("/validate/{element}", name="validatePerson")
      * @Method({"POST"})
+     * @param Request $request
+     * @param $element
+     * @return JsonResponse
      */
     public function validate(Request $request, $element)
     {
         try {
-            $input = json_decode($request->getContent(), true)['input'];
-        } catch (\Exception $e) {
+            $input = $input = json_decode($request->getContent(), true)['input'];
```

Comment:
Už tokį kodą galėtų ir nagus nukapoti :laughing: 
```suggestion
            $input = json_decode($request->getContent(), true)['input'];
```

Tai veikia, nes
```php
echo ($a = '123'); // 123
echo $a; // 123
echo ($a = ($a = '123')); // 123
```
T.y. PHP kalboje priskirimas irgi duodą rezultatą.
Bet tai kitiems kolegoms sakytų, kad nesuvoki programavimo kalbos principų, nes priskyrimo rezultatas naudojamas tik:
```php
if ($normalValue = someFunction()) {
 // Do something iwth $normalValue
} else {
// $normalValue is false or null
}
```
Ir tai jau laikoma bloga praktika, nes IDE nežino, ar specialiai parašei `=`, o gal suklydai ir norėjai `==`.

Plačiau:
* http://php.net/manual/en/language.operators.assignment.php